autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingHandler.java

```diff
@@ -48,30 +50,37 @@
 
     @Override
     public void onApplicationEvent(ScalingEvent event) {
-        BaseAlert alert = event.getAlert();
-        Cluster cluster = clusterService.findById(alert.getCluster().getId());
+        Set<BaseAlert> alerts = event.getAlerts();
+        Long clusterId = alerts.stream().findFirst().map(ma -> ma.getCluster().getId()).orElseThrow();
+        Cluster cluster = clusterService.findById(clusterId);
         MDCBuilder.buildMdcContext(cluster);
-        scale(cluster, alert.getScalingPolicy());
+        if (isCooldownElapsed(cluster)) {
+            alerts.forEach(alert -> scale(cluster, alert.getScalingPolicy()));
```

Comment:
Do we really want to trigger multiple scaling actions at the same time?

Reply:
Yes, this is the main problem.

Reply:
Hm, I need more investigation. The second request will be denied because the cluster is under operation.

Reply:
I am not sure about that, triggering multiple scaling on the CB APIs is not so safe. @keyki ?